src/test/java/com/configcat/ConfigCatClientTest.java

```diff
@@ -810,18 +852,19 @@ void testOnFlagEvaluationError() throws IOException {
     }
 
     @Test
-    void testCacheKey() throws NoSuchFieldException, IllegalAccessException {
-        //Test Data: SDKKey "test1", HASH "147c5b4c2b2d7c77e1605b1a4309f0ea6684a0c6"
-        ConfigCatClient clTest1 = ConfigCatClient.get("test1");
+    void testCacheKey() throws NoSuchFieldException, IllegalAccessException, IOException {
```

Comment:
I can rewrite the test not to use the client, just the ConfigService, so the sdkKey will not be validated. But to be honest, in the long term, I think the test should use valid SDK keys as input data.

I guess for now, it would be easier if I changed the tests. What do you think?

Reply:
Resolved in private discussion: it'd be better to test with valid SDK keys, so we'd rather update these tests in other SDKs. (I already did that in the .NET SDK, the expected hashes match.)